src/account/components/AccountLogin.js

```diff
@@ -10,10 +11,15 @@ import PageLoader from 'common/components/PageLoader';
 
 import logo from 'assets/logo.svg';
 
+const appendReferrer = (url, referrer) =>
+  referrer ? `${url}&state=${referrer}` : url;
```

Comment:
issue: use a more descriptive variable name than `state`

Reply:
@paulschreiber This was decided on the backend side, for context:
- Bakend PR: https://github.com/techmatters/terraso-backend/pull/116
- Slack conversation: https://tech-matters.slack.com/archives/C02K02L3Q48/p1643385309308109
Tagging @cacarrara for vis

Reply:
@paulschreiber I'm ok with `state`. This is the default variable name for this purpose.